### PR TITLE
Refactor Player and Tile class and add some useful functions and props

### PR DESCRIPTION
**Pull request by maxmeijer42**

### CODE REVIEW

File: MeesGame/Gameplay/Level/Level.cs [hunk 1]
```diff
@@ -25,14 +25,16 @@ public Level(int levelindex = 0)
             {
                 for (int y = 0; y < numRows; y++)
                 {
-                    tiles.Add(new FloorTile(), x, y);
+                    Tile tile = new FloorTile();
+                    tiles.Add(tile, x, y);
+                    tile.Location = new Point(x, y);
```
Comment (maxmeijer42) [inline MeesGame/Gameplay/Level/Level.cs]: Dit gebeurt al in de TileField klasse. Ik zal het aanpassen

**Review by jensj12 — APPROVED** [review]

**Review by jensj12 — DISMISSED** [review]
Contains TODO's, breaks doors (which already crashed anyways, but this reduces functionality even further)

**Comment by maxmeijer42** [comment]
The TODO's were already there. One todo is for an issue not assigned to me. The TODO's were added in PR #43. Doors already crash but storing player history is not the way to check if a player has picked up a key. Code breaks nothing, adds Inventory which can store keys. @LisanneW might want to use that to store Keys.

**Review by LisanneW — CHANGES_REQUESTED** [review]
Please format files using ctrl+k+d, at least level.cs needs it.

**Comment by maxmeijer42** [comment]
Indeed there was a trailing tab in level.cs. It's fixed now.

**Review by jensj12 — APPROVED** [review]

**Review by LisanneW — APPROVED** [review]

**Pull request merged**